app/controllers/PhotographExternalSealController.scala

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers
+
+import controllers.actions.*
+import models.{ArrivalId, NormalMode}
+import play.api.i18n.{I18nSupport, MessagesApi}
+import play.api.mvc.{Action, AnyContent, Call, MessagesControllerComponents}
+import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendBaseController
+import views.html.PhotographExternalSealView
+
+import java.time.Year
+import javax.inject.Inject
+
+class PhotographExternalSealController @Inject() (
+  override val messagesApi: MessagesApi,
+  val controllerComponents: MessagesControllerComponents,
+  actions: Actions,
+  view: PhotographExternalSealView
+) extends FrontendBaseController
+    with I18nSupport {
+
+  private val expiryYear = Year.now().getValue + 3
```

Comment:
We could move this method to the DateTimeService and mock the response.

Also is it + 3 or + 4?

> for the remaining year, then a further 3 full years

That would mean, for the example of today, the rest of 2024, then 2025, 2026 and 2027. So would it not be 1st January **2028**?

Reply:
Ah, that makes sense after reading it again, will do a quick check with Gavin again though.